src/gasman.c

```diff
@@ -405,6 +417,18 @@ Bag                     ChangedBags;
 */
 static Bag MarkedBags;
 
+/*  These are used to overwrite masterpointers which may still be
+linked from weak pointer objects but whose bag bodies have been
+collected.  Two values are used so that old masterpointers of this
+kind can be reclaimed after a full garbage collection. The values must
+not look like valid pointers, and should be congruent to 1 mod sizeof(Bag),
+to ensure that IS_WEAK_DEAD_BAG works correctly.
+*/
+
+Bag * NewWeakDeadBagMarker = (Bag *)(1000*sizeof(Bag) + 1L);
+Bag * OldWeakDeadBagMarker = (Bag *)(1001*sizeof(Bag) + 1L);
```

Comment:
Why is this moved around?

Reply:
I need to refer to it in an earlier place in the file

Reply:
But turns out I didn't end up using that. Taken this part of the PR out.